test/controllers/foreman/registration_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'katello_test_helper'
+
+class RegistrationControllerTest < ActionController::TestCase
```

Comment:
We tend to not write new tests in the spec style. do you mind changing these to unit style?

Reply:
I'd also like to see a test added asserting that the blank activation key field is handled properly.

Reply:
> We tend to not write new tests in the spec style. do you mind changing these to unit style?

Changed.

> I'd also like to see a test added asserting that the blank activation key field is handled properly.

Its there:
https://github.com/Katello/katello/pull/9012/files#diff-d17fa6ea78f8ea3cee9a2327b2c863311d7e9fa82064069c714a4ab1b1cc35ecR42-R51